main/src/ca/uwaterloo/flix/language/dbg/printer/MonoAstPrinter.scala

```diff
@@ -47,7 +47,7 @@ object MonoAstPrinter {
     case Expr.Cast(exp, Some(declaredType), _, _, _, _) => DocAst.Expr.Cast(print(exp), TypePrinter.print(declaredType))
     case Expr.Cast(_, _, _, _, _, _) => DocAst.Expr.Unknown
     case Expr.TryCatch(exp, rules, _, _, _) => DocAst.Expr.TryCatch(print(exp), rules.map(printCatchRule))
-    case Expr.TryWith(exp, _Use, rules, _, _, _) => DocAst.Expr.TryWith(print(exp), _Use.sym, rules.map(printHandlerRule))
+    case Expr.TryWith(exp, effUse, rules, _, _, _) => DocAst.Expr.RunWithHandler(print(exp), effUse.sym, rules.map(printHandlerRule))
```

Comment:
effUse -> symUse?

Reply:
I have not touched the AST's after the desugaring on purpose in this PR to limit the scope. I will fix those and rename to run in the next PR